modules/service/src/main/scala/higherkindness/mu/rpc/channel/ManagedChannelInterpreter.scala

```diff
@@ -25,7 +26,9 @@ import io.grpc._
 
 class ManagedChannelInterpreter[F[_]](
     initConfig: ChannelFor,
-    configList: List[ManagedChannelConfig]
+    configList: List[ManagedChannelConfig],
+    builderForAddress: (String, Int) => ManagedChannelBuilder[_] = ManagedChannelBuilder.forAddress,
+    builderForTarget: String => ManagedChannelBuilder[_] = ManagedChannelBuilder.forTarget
```

Comment:
It is source-compatible because of the default arguments, but it is binary-incompatible: https://github.com/jatcwang/binary-compatibility-guide#dont-adding-parameters-with-default-values-to-methods

I'll leave the primary constructor alone and add a secondary constructor for use in tests.